app/helpers/view_helper.rb

```diff
@@ -122,6 +122,30 @@ def protect_against_mistakes(anchor:)
     end
   end
 
+  UTM_PARAMS = {
```

Comment:
i think these methods could live in their own helper module, eg `utm_link_helper.rb`